sound/soc/sof/control.c

```diff
@@ -137,9 +136,7 @@ int snd_sof_volume_put(struct snd_kcontrol *kcontrol,
 	/* notify DSP of mixer updates */
 	if (pm_runtime_active(scomp->dev))
 		snd_sof_ipc_set_get_comp_data(scontrol,
-					      SOF_CTRL_TYPE_VALUE_CHAN_SET,
-					      SOF_CTRL_CMD_VOLUME,
-					      true);
+					      SOF_CTRL_TYPE_VALUE_CHAN_SET, true);
```

Comment:
typo in commit message:  s/scrontrol/scontrol

Reply:
OK

Reply:
Maybe also correct "duplication information" to "duplicated information"

Reply:
The commit messages went through several iterations and you are right, I'll fix this up as well.
Thanks!